crates/aptos/src/node/local_testnet/docker.rs

```diff
@@ -30,7 +30,8 @@ pub const CONTAINER_NETWORK_NAME: &str = "aptos-local-testnet-network";
 /// there no second location, there is just the one named pipe.
 pub async fn get_docker() -> Result<Docker> {
     let docker = Docker::connect_with_local_defaults()
-        .context(format!("{} (init_default)", ERROR_MESSAGE))?;
+        .context(format!("{} (init_default)", ERROR_MESSAGE))
+        .inspect_err(|e| eprintln!("{:#}", e))?;
```

Comment:
Just confirming, you're happy for this to be spread across multiple lines? If not, you can just use `{}` instead of `{:#}`.

Reply:
looks like it is one line?
```
Unexpected error: Failed to apply pre run steps for Postgres: Docker is not available, confirm it is installed and running. See https://aptos.dev/guides/local-development-network#faq for assistance. (version_home): error trying to connect: No such file or directory (os error 2): No such file or directory (os error 2)
```
<img width="1743" alt="image" src="https://github.com/user-attachments/assets/9e8c9853-594d-4d1f-b183-5fbe24e188a0">